apps/drec-ui/src/pages/PageNotFound/PageNotFound.tsx

```diff
@@ -0,0 +1,24 @@
+import { Button, Paper, Typography } from '@material-ui/core';
+import React, { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+import { useStyles } from './PageNotFound.styles';
+
+export const PageNotFound: FC = () => {
```

Comment:
Why don't you use the one from ui-core package? I guess it is the same there

Reply:
The padding breaks the window size. Here I just added padding top and bottom

Reply:
Well noted, will fix this in `ui-core`. 
Lets keep it here for now